src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
+                        "io_getevents\0"
+                        "io_setup\0"
+                        "io_submit\0"
+                        "ioctl\0"
+                        "ioprio_get\0"
+                        "ioprio_set\0"
+                        "lgetxattr\0"
+                        "link\0"
+                        "linkat\0"
+                        "listxattr\0"
+                        "llistxattr\0"
+                        "lookup_dcookie\0"
+                        "lremovexattr\0"
+                        "lseek\0"
+                        "lsetxattr\0"
+                        "lstat64\0"
+                        "lstat\0"
+                        "memfd_create\0"
+                        "mkdir\0"
+                        "mkdirat\0"
+                        "mknod\0"
+                        "mknodat\0"
+                        "mmap2\0"
+                        "mmap\0"
+                        "name_to_handle_at\0"
+                        "newfstatat\0"
+                        "oldfstat\0"
+                        "oldlstat\0"
+                        "oldstat\0"
+                        "open\0"
+                        "open_by_handle_at\0"
+                        "openat\0"
+                        "poll\0"
+                        "ppoll\0"
+                        "pread64\0"
+                        "preadv\0"
+                        "process_vm_readv\0"
+                        "process_vm_writev\0"
+                        "pselect6\0"
+                        "pwrite64\0"
+                        "pwritev\0"
+                        "read\0"
+                        "readahead\0"
+                        "readdir\0"
+                        "readlink\0"
+                        "readlinkat\0"
+                        "readv\0"
+                        "removexattr\0"
+                        "rename\0"
+                        "renameat2\0"
+                        "renameat\0"
+                        "rmdir\0"
+                        "select\0"
+                        "setxattr\0"
+                        "splice\0"
+                        "stat64\0"
+                        "stat\0"
+                        "statfs64\0"
+                        "statfs\0"
+                        "symlink\0"
+                        "symlinkat\0"
+                        "sync\0"
+                        "sync_file_range2\0"
+                        "sync_file_range\0"
+                        "syncfs\0"
+                        "tee\0"
+                        "truncate64\0"
+                        "truncate\0"
+                        "umask\0"
+                        "unlink\0"
+                        "unlinkat\0"
+                        "userfaultfd\0"
+                        "ustat\0"
+                        "utime\0"
+                        "utimensat\0"
+                        "utimes\0"
+                        "vmsplice\0"
+                        "write\0"
+                        "writev\0",
```

Comment:
I am pretty sure that select/poll/eventfd/epoll should really be split out of this. i.e. i'd have one group that is about actually file system access, and the other that is just about event loop stuff. stuff like memfd/userfaultd/splice/tree/ioprio_set/ioprio_get should probably left out entirely because it belongs to neither...